docs/index.md

```diff
@@ -1,57 +1,7 @@
-
-1. Swift onboarding
-
-2. Data Security
-
-3. User Privacy
-
-4. Payment Security
-
-5. User Experience
-
-6. Customer Experience
-
-7. Developer Experience
-
-8. App Performance
-
-9. Affordability
-
-**What is launched in Ultra V1?**
-
-In Ultra V1, we have setup a minimal viable process required to onboard a partner keeping above ethos in check. We also took some strategic product decisions which has shaped Ultra as a service offering to partners, like following
-
-1. User experience over purchase funnel will be built and owned by the partner.
-
-2. Discovery of Applications will be controlled by Flipkart.
-
-3. Only Logged in Users can access partner applications.
-
-4. Payments is powered by Flipkart Payment Gateway(FKPG).
-
-5. Flipkart can run offers on partners purchase funnel.
-
-6. Fulfillment and post purchase experience will also be built and owned by the partner.
-
-7. All customer support related queries will be answered by the partner
-
-8. Money will reach partner’s nodal account directly.
-
-You can read step by step approach to onboard a partner in the Onboarding section
-
-**What is in for Partners/Businesses?**
-
-Partners can integrate their Apps to Flipkart’s Android App using developer’s friendly API’s and use the benefit of being on more than 100 million handhelds in the country. 
-
-**What is in for Developers?**
-
-Ultra plans to become one of the most dominant channel that no business or, developer can afford to miss. Ultra platform is one of its kind which supports a wide variety of technologies from the most basic web apps to very advanced experiences powered by React Native. This provides ample room to grow and develop unique skills while keeping entry barrier low so that no one is overwhelmed. In near future we would want Ultra to be a prestigious developer platform enabling cutting edge experiences. Being great on Ultra should be synonymous with being a great developer.
-
+The process involves on-boarding partners who specialise in these categories. It redirects users to partner experiences within the Flipkart app and grants the entry into these experiences based on Flipkart user login and makes payments for these orders using Flipkart’s payment gateway. If users want to check the status of these orders, they can do that using Flipkart’s My Orders page.
 
+This will help partners and businesses by allowing them to integrate their applications within Flipkart’s android application using developer friendly APIs and open doors to over 100 million handhelds in the country. Also, developers would not want to miss a chance of working with Ultra platform as it supports a wide variety of technologies from the most basic web applications to very advanced experiences powered by React Native. This provides ample room to grow and develop unique skills while keeping entry barrier low such that it’s not overwhelming. The vision of Ultra is to be a prestigious developer platform enabling cutting edge experiences to all.
```

Comment:
This looks good, but can you add the rest ?

Reply:
Added more